services/ui-src/src/components/rates/types/Fields.tsx

```diff
@@ -187,8 +177,10 @@ export const Fields = (props: PageElementProps<LengthOfStayRateTemplate>) => {
         <CmsdsTextField
           label={labels.adjustedRate}
           name="adjustedRate"
+          onChange={onChangeHandler}
+          onBlur={onChangeHandler}
           value={displayValue.adjustedRate}
-          disabled={true}
+          disabled={disabled}
```

Comment:
what's the difference between disabled={true} vs disabled={disabled} is that the same functionality? This is the one that used to be auto calculated right?